### PR TITLE
Add new RewardQuest table structure

### PR DESCRIPTION
**Pull request by DigitalBox98**
Add new RewardQuest table structure with one example quest

### CODE REVIEW

File: data/_TableSchemas.json [hunk 1]
```diff
@@ -7193,6 +7244,157 @@
       }
     ]
   },
+  {
+    "Name": "RewardWuest",
```
Comment (LRFalk01) [inline data/_TableSchemas.json:7248]: `Wuest` Should this be `Quest`

Comment (DigitalBox98) [reply data/_TableSchemas.json:7248]: Nice catch. I will fix this typo error :)

Comment (DigitalBox98) [reply data/_TableSchemas.json]: Fixed:)

**Review by LRFalk01 — APPROVED** [review]

**Pull request merged**